File: Python/1145.py
x,y = map(int,input().split())

n = []
i = 0
k = 0
while i < y-x:
    for j in range(1, x + 1):
        n.append(i + 1)
        aux = n[k]
        n[k] = str(n[k])
        k = k + 1
        i = i + 1
    
    k = 0
    n = ' '.join(n)
    print(n)
    n =[]
z = y%x

if z ==0:
    for w in range(y - x,y):
        n.append(w + 1)
        n[k] = str(n[k])
        k = k + 1
    n = ' '.join(n)
    print(n)
if z !=0:
    for w in range(aux,y):
        n.append(w + 1)
        n[k] = str(n[k])
        k = k + 1
    n = ' '.join(n)
    print(n)

File: CSharp/1145.cs
using System;
using System.Text;
using System.Linq;

class URI
{
    static void Main()
    {
        int[] a = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int x = a[0];
        int y = a[1];

        int aux = 1;

        while (aux <= y)
        {
            StringBuilder print = new StringBuilder();
            for (int i = 0; i < x && aux <= y; i++)
            {
                if (i > 0)
                {
                    print.Append(" ");
                }
                print.Append(aux);
                aux++;
            }
            Console.WriteLine(print.ToString());
        }
    }
}
